src/ipa/rpi/vc4/data/ov64a40.json

```diff
@@ -0,0 +1,427 @@
+        {
+            "rpi.lux":
+            {
+                "reference_shutter_speed": 17861,
+                "reference_gain": 2.0,
+                "reference_aperture": 1.0,
+                "reference_lux": 1073,
+                "reference_Y": 9022
+            }
+        },
+        {
+            "rpi.noise":
+            {
+                "reference_constant": 0,
+                "reference_slope": 2.984
+            }
+        },
+        {
+            "rpi.geq":
+            {
+                "offset": 215,
+                "slope": 0.01121
+            }
+        },
+        {
+            "rpi.sdn": { }
+        },
+        {
+            "rpi.awb":
+            {
+                "priors": [
+                    {
+                        "lux": 0,
+                        "prior":
+                        [
+                            2000, 1.0,
+                            3000, 0.0,
+                            13000, 0.0
+                        ]
+                    },
+                    {
+                        "lux": 800,
+                        "prior":
+                        [
+                            2000, 0.0,
+                            6000, 2.0,
+                            13000, 2.0
+                        ]
+                    },
+                    {
+                        "lux": 1500,
+                        "prior":
+                        [
+                            2000, 0.0,
+                            4000, 1.0,
+                            6000, 6.0,
+                            6500, 7.0,
+                            7000, 1.0,
+                            13000, 1.0
+                        ]
+                    }
+                ],
+                "modes":
+                {
+                    "auto":
+                    {
+                        "lo": 2500,
+                        "hi": 8000
+                    },
+                    "incandescent":
+                    {
+                        "lo": 2500,
+                        "hi": 3000
+                    },
+                    "tungsten":
+                    {
+                        "lo": 3000,
+                        "hi": 3500
+                    },
+                    "fluorescent":
+                    {
+                        "lo": 4000,
+                        "hi": 4700
+                    },
+                    "indoor":
+                    {
+                        "lo": 3000,
+                        "hi": 5000
+                    },
+                    "daylight":
+                    {
+                        "lo": 5500,
+                        "hi": 6500
+                    },
+                    "cloudy":
+                    {
+                        "lo": 7000,
+                        "hi": 8600
+                    }
+                },
+                "bayes": 1,
+                "ct_curve":
+                [
+                    2300.0, 1.0522, 0.4091,
```

Comment:
2300 is an admirably low value, might your incandescent mode want to stretch down this far? (Currently it stops at 2500.)

Reply:
At present, the lowest level of our light source is 2300k and the next level is 2700k. We expect it to be able to interpolate to a 2500k light source. If 2300k is deleted, it will start from 2700k. Will there be any negative impact from this?

Reply:
> No - just that you have no way to get a colour temperature lower than 2500. But if you're happy for the AWB searches to stop at 2500, then that's fine and we are happy to merge this as it is!

Got it, that's acceptable to me.